apps/com.thesagaz/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Sagaz",
+    "href": "https://thesagaz.com",
+    "desc": "Caelum AB Inferno",
+    "category": "collectibles",
+    "tags": [
+        "Sagaz",
+        "VeChain",
+        "VIP-181",
+        "Collectibles",
+        "Art",
+        "NFT",
+        "PFP",
+        "Sassafras",
+        "SASS",
+        "Questing",
+        "Battles",
+        "Training"
+    ],
+    "contracts": [
+        "0xf46d132cfc813da70c0806921d05ab3057d8badd",
+        "0x88270ca93379de8245d653070d958ae28b91B01c",
```

Comment:
The contract address should be in lowercase.

Reply:
Oh man I thought i got them in lower case one sec man I really sure am having a go with this lol I am so sorry for maken you come back to this over and over I feel bad